native/core/src/store.rs

```diff
@@ -13,16 +14,16 @@ use crate::{
 pub struct Store {
     client: Client,
     fresh: bool,
-    models: Arc<dashmap::DashMap<String, AnyActerModel>>,
-    indizes: Arc<dashmap::DashMap<String, Vec<String>>>,
-    dirty: Arc<dashmap::DashSet<String>>,
+    models: Arc<DashMap<String, AnyActerModel>>,
+    indizes: Arc<DashMap<String, Vec<String>>>,
+    dirty: Arc<DashSet<String>>,
 }
 
 static ALL_MODELS_KEY: &str = "ACTER::ALL";
 static DB_VERSION_KEY: &str = "ACTER::DB_VERSION";
 static CURRENT_DB_VERSION: u32 = 1;
 
-async fn get_from_store<T: serde::de::DeserializeOwned>(client: Client, key: &str) -> Result<T> {
+async fn get_from_store<T: DeserializeOwned>(client: Client, key: &str) -> Result<T> {
```

Comment:
It is common practice to keep the prefix `serde::de` to make apparent what this is (it's `serde` and not some local thing). I'd really like us to keep those please.
```suggestion
async fn get_from_store<T: serde::de::DeserializeOwned>(client: Client, key: &str) -> Result<T> {
```